db/settings.go

```diff
@@ -5,10 +5,22 @@ type Settings struct {
 	Token     string
 }
 
+type Getchatid struct {
+	chatId int64
```

Comment:
Also, set() and get() functions are in case the key's value contains more than one parameter. Please use setRaw() and getRaw() for single parameter value without any struct.

Reply:
Ok